D1371R3.md

```diff
@@ -807,13 +808,11 @@ inspect (v) {
 
 The case pattern has the form:
 
-> | `case` _pattern_
+> | `case` _expression-pattern_
```

Comment:
Note that I defined case pattern in terms of expression pattern, which in the end expects constant expression.
And if I read http://eel.is/c++draft/expr.const correctly should not have issue with parsing `,`.